### PR TITLE
#9 update CI configuration

### PR DESCRIPTION
**Pull request by CAMOBAP795**

### CODE REVIEW

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -26,38 +25,10 @@ jobs:
...
-          python-version: '2.7'
-          architecture: 'x64'
-      - name: Install xml2rfc
-        run: |
-          pip install xml2rfc
-      - name: Metanorma compile
-        shell: pwsh
+          gem install bundler
+          bundle config --local path vendor/bundle
+          bundle update
```
Comment (ronaldtse) [inline .github/workflows/windows.yml]: @CAMOBAP795 There's no `Gemfile.lock` so there's no need to `bundle update`?

Comment (CAMOBAP795) [reply .github/workflows/windows.yml]: Good catch! should we remove it from all CI configuration, AFAIR we decided to not commit `Gemfile.lock` anymore, right?

Comment (ronaldtse) [reply .github/workflows/windows.yml]: Yes please! Let's get of them :wink:

Comment (CAMOBAP795) [reply .github/workflows/windows.yml]: I will https://github.com/metanorma/metanorma-build-scripts/issues/36

**Pull request merged**